### PR TITLE
Cwarren5701/xp points to multichoice questions

### PR DESCRIPTION
**Pull request by cwarren5701**
Fixed the bug and added a box for to display the XP to be rewarded.

### CODE REVIEW

File: IntroToCSharp/Shared/Components/Activities/MultipleChoiceQuestion.razor [hunk 1]
```diff
@@ -110,8 +128,9 @@
     {
         this._options.ForEach(option => option.IsChecked = option.IsCorrect);
         this.SelectedOption = this._options.First(option => option.IsCorrect).Text;
-        this.Submit();
+        // We had to move this line above so that the submit method can know when the user clicked "show answer" and then award xp accordingly.
```
Comment (jcollard) [inline IntroToCSharp/Shared/Components/Activities/MultipleChoiceQuestion.razor:131]: No need to put this comment here.

**Review by jcollard — CHANGES_REQUESTED** [review]
Looks good. Just need to take out the one comment.

File: IntroToCSharp/Shared/Components/Activities/MultipleChoiceQuestion.razor (hunk 1 above)
Comment (cwarren5701) [reply IntroToCSharp/Shared/Components/Activities/MultipleChoiceQuestion.razor]: done

**Comment by jcollard** [comment]
@josht77 @cwarren5701 there is a merge conflict. You'll need to merge develop into the branch to resolve it.

File: IntroToCSharpSite.code-workspace [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+	"folders": [
+		{
+			"path": "."
+		},
+		{
+			"path": "IntroToCSharp/wwwroot/examples/csharp"
+		}
+	],
+	"settings": {}
+}
```
Comment (jcollard) [inline IntroToCSharpSite.code-workspace:11]: This file should not be added. You should either delete it locally or add it to .gitignore.

File: IntroToCSharp/Shared/Components/Activities/MultipleChoiceQuestion.razor [hunk 3]
```diff
@@ -75,14 +82,15 @@
     private string SelectedOption { get; set; }
     private bool _isSubmitted = false;
     private bool _isWrong = false;
-    private int _attempts = 0;
+    private int attempts = 0;
```
Comment (jcollard) [inline IntroToCSharp/Shared/Components/Activities/MultipleChoiceQuestion.razor:85]: private fields should start with an underscore.

File: IntroToCSharp/Shared/Components/Activities/MultipleChoiceQuestion.razor [hunk 4]
```diff
@@ -75,14 +82,15 @@
...
 
     private string ExplanationStyle = $"background: #ffffe3; color: {Themes.s_CSharpTheme.Palette.Black.ToString()};";
+    private string XPDisplayStyle = $"color: {Themes.s_CSharpTheme.Palette.Success.ToString()};";
```
Comment (jcollard) [inline IntroToCSharp/Shared/Components/Activities/MultipleChoiceQuestion.razor:88]: Private fields should start with an underscore.

**Review by jcollard — CHANGES_REQUESTED** [review]
A few naming things. See: https://www.c-sharpcorner.com/UploadFile/8a67c0/C-Sharp-coding-standards-and-naming-conventions/

**Review by jcollard — APPROVED** [review]
lgtm

**Pull request merged**